iOS/Swift/README.md

```diff
@@ -152,8 +152,47 @@ else {
 
 When they are needed, use comments to explain **why** a particular piece of code does something. Comments must be kept up-to-date or deleted.
 
-Avoid block comments inline with code, as the code should be as self-documenting as possible. *Exception: This does not apply to those comments used to generate documentation.*
+Avoid block comments inline with code, as the code should be as self-documenting as possible. When you put a comment inside a function this is an indication you could make it a function. Like so:
 
+```swift
+  func isFooFunny (foo : Foo) throws -> Bool? {
```

Comment:
According to our coding guidelines you should remove the space before the `(` and before the `:` like this:

```
func isFooFunny(foo: Foo) throws -> Bool? {
```